playbooks/inventory.yml

```diff
@@ -1,10 +1,7 @@
 ---
 all:
   hosts:
-    dev:
-      ansible_host: 127.0.0.1
-      ansible_connection: local
-  children:
-    vagrant:
-      hosts:
-        dev:
+    acid-server:
+      ansible_host:
+      ansible_user: "ubuntu"
+      ansible_python_interpreter: /usr/bin/python3
```

Comment:
Not a good idea. Not all ansible modules work with python3, this is why whole world still uses python2 for Ansible.